src/main/java/com/squareup/javapoet/JavaFile.java

```diff
@@ -83,6 +84,11 @@ public void writeTo(Appendable out) throws IOException {
 
   /** Writes this to {@code directory} as UTF-8 using the standard directory structure. */
   public void writeTo(Path directory) throws IOException {
+    writeTo(directory, UTF_8);
+  }
+
+  /** Writes this to {@code directory} with the provided {@code charset} using the standard directory structure. */
```

Comment:
Looks like checkstyle is complaining that this line is longer than the limit, which is 100 lines. Can you please fix?

Reply:
I corrected this. Sorry that I not see this in the first place.

Reply:
Thanks for fixing!